test/functional/npcontainer.spec.ts

```diff
@@ -19,8 +19,8 @@ const legacyClient = new CosmosClient({
         return next(context);
       }
     }
-  ]
-});
+  ] as PluginConfig[]
```

Comment:
this doesn't seem ideal. Couldn't we have new'd up the config on a separate line, then asset it to any and add the plugins?

```typescript
const options: CosmosClientOptions = {  /* stuff */ };
const plugins: PluginConfig = { /* stuff */ };
new CosmosClient({...options, plugins} as any);
```

That way we'd still get type help on the other settings in CosmosClientOptions, right?

Reply:
Yes. That is better